src/pages/index.js

```diff
@@ -34,29 +30,76 @@ const gallerySection = new Section(
   ".galery__cards"
 );
 
+modalsForms.forEach((form) => {
+  formvalidatorList[form.id] = new FormValidator(optionsValidation, form);
+  formvalidatorList[form.id].enableValidation();
+});
+
 const userInfo = new UserInfo({
   nameSelector: ".profile__title",
   jobSelector: ".profile__subtitle",
+  avatarSelector: ".profile__image",
 });
 
 const profileModal = new PopupWithForm(
   "#modal-profile",
   (formData) => {
-    userInfo.setUserInfo(formData.name, formData.job);
+    api
+      .editProfile({ name: formData.name, about: formData.job })
+      .then((responce) => {
+        userInfo.setUserInfo(responce.name, responce.about);
+        profileModal.resetFormAndClose();
+      })
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**